discoverer/src/main/java/eu/seaclouds/platform/discoverer/core/UniqueFileGen.java

```diff
@@ -32,9 +34,13 @@
 public class UniqueFileGen {
```

Comment:
This class was already present inside the Discoverer and was used to generate offering files.
Since I had to focus on other things I didn't tried to refactor it.